queries/ipkg/highlights.scm

```diff
@@ -0,0 +1,53 @@
+; https://github.com/srghma/tree-sitter-ipkg
+; Copyright (c) 2024 Serhii Khoma
+; Licensed under the MIT license.
```

Comment:
Please don't add license information here; all queries are automatically MIT licensed through the repo.